lib/bap_llvm/llvm_disasm.cpp

```diff
@@ -615,14 +615,30 @@ class llvm_disassembler : public disassembler_interface {
     }
 };
 
+
+bool is_error_prone_arch(const char *triple) {
+#if LLVM_VERSION_MAJOR >= 4 && LLVM_VERSION_MAJOR < 8
```

Comment:
don't use macro here. If anything could be done without a macro, it should be done without it.